layouts/shortcodes/cve-feed.html

```diff
@@ -1,9 +1,15 @@
 {{ $feed := getJSON .Site.Params.cveFeedBucket }}
+{{ if eq $feed nil }}
+  {{ warnf "Cannot load CVE feed." }}
```

Comment:
Please check if this is a production build, and error out of it is and there's no feed.
```suggestion
  {{- if eq hugo.Environment "production" }}
    {{ errorf "Unable to load CVE feed. Aborting build." }}
  {{- else -}}
    {{ warnf "Cannot load CVE feed." }}
  {{- end -}}